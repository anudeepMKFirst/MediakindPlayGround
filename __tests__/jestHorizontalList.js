import React from 'react';
import HorizontalList from "./../horizontallist.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<HorizontalList/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});