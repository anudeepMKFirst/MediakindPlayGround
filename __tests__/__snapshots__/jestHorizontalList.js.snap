// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Composition
  onTimelinesLoaded={[Function]}
  source="Lander_Horizontal-Container"
>
  <ListRef
    DEPRECATED_sendUpdatedChildFrames={false}
    alwaysBounceHorizontal={true}
    alwaysBounceVertical={false}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={true}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    name="List-Horizontal"
    numColumns={1}
    onCompositionDidLoad={[Function]}
    onContentSizeChange={null}
    onEndReachedThreshold={2}
    onLayout={[MockFunction]}
    onMomentumScrollBegin={[MockFunction]}
    onMomentumScrollEnd={[MockFunction]}
    onResponderGrant={[MockFunction]}
    onResponderReject={[MockFunction]}
    onResponderRelease={[MockFunction]}
    onResponderTerminationRequest={[MockFunction]}
    onScroll={[MockFunction]}
    onScrollBeginDrag={[MockFunction]}
    onScrollEndDrag={[MockFunction]}
    onScrollShouldSetResponder={[MockFunction]}
    onStartShouldSetResponder={[MockFunction]}
    onStartShouldSetResponderCapture={[MockFunction]}
    onTimelinesLoaded={[Function]}
    onTouchCancel={[MockFunction]}
    onTouchEnd={[MockFunction]}
    onTouchMove={[MockFunction]}
    onTouchStart={[MockFunction]}
    renderItem={[Function]}
    renderScrollComponent={[Function]}
    scrollBarThumbImage={null}
    scrollEnabled={true}
    scrollEventThrottle={50}
    sendMomentumEvents={true}
    stickyHeaderIndices={Array []}
    style={
      Array [
        Object {
          "flexDirection": "row",
          "flexGrow": 1,
          "flexShrink": 1,
          "overflow": "scroll",
        },
        undefined,
      ]
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <ScrollContentData
      collapsable={false}
      onLayout={[MockFunction]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          undefined,
        ]
      }
    />
  </ListRef>
  <ViewRef
    name="List-Horizontal"
    onCompositionDidLoad={[Function]}
    onTimelinesLoaded={[Function]}
  />
</Composition>
`;
